pkg/assembler/backends/ent/schema/certification.go

```diff
@@ -61,9 +62,9 @@ func (Certification) Edges() []ent.Edge {
 
 func (Certification) Indexes() []ent.Index {
 	return []ent.Index{
-		index.Fields("type", "justification", "origin", "collector", "source_id", "known_since").Unique().Annotations(entsql.IndexWhere("source_id IS NOT NULL AND package_version_id IS NULL AND package_name_id IS NULL AND artifact_id IS NULL")),
-		index.Fields("type", "justification", "origin", "collector", "package_version_id", "known_since").Unique().Annotations(entsql.IndexWhere("source_id IS NULL AND package_version_id IS NOT NULL AND package_name_id IS NULL AND artifact_id IS NULL")),
-		index.Fields("type", "justification", "origin", "collector", "package_name_id", "known_since").Unique().Annotations(entsql.IndexWhere("source_id IS NULL AND package_version_id IS NULL AND package_name_id IS NOT NULL AND artifact_id IS NULL")),
-		index.Fields("type", "justification", "origin", "collector", "artifact_id", "known_since").Unique().Annotations(entsql.IndexWhere("source_id IS NULL AND package_version_id IS NULL AND package_name_id IS NULL AND artifact_id IS NOT NULL")),
+		index.Fields("type", "justification", "origin", "collector", "document_ref", "source_id", "known_since", "document_ref").Unique().Annotations(entsql.IndexWhere("source_id IS NOT NULL AND package_version_id IS NULL AND package_name_id IS NULL AND artifact_id IS NULL")),
+		index.Fields("type", "justification", "origin", "collector", "document_ref", "package_version_id", "known_since", "document_ref").Unique().Annotations(entsql.IndexWhere("source_id IS NULL AND package_version_id IS NOT NULL AND package_name_id IS NULL AND artifact_id IS NULL")),
+		index.Fields("type", "justification", "origin", "collector", "document_ref", "package_name_id", "known_since", "document_ref").Unique().Annotations(entsql.IndexWhere("source_id IS NULL AND package_version_id IS NULL AND package_name_id IS NOT NULL AND artifact_id IS NULL")),
+		index.Fields("type", "justification", "origin", "collector", "document_ref", "artifact_id", "known_since", "document_ref").Unique().Annotations(entsql.IndexWhere("source_id IS NULL AND package_version_id IS NULL AND package_name_id IS NULL AND artifact_id IS NOT NULL")),
```

Comment:
It looks like `document_ref` is repeated in indexes creations.

Reply:
oops good catch. I will fix.

Reply:
done